snippets/ztp-example-siteconfig.adoc

```diff
@@ -77,7 +77,7 @@ spec:
 <4> Cluster labels must correspond to the `bindingRules` field in the `PolicyGenTemplate` CRs that you define. For example, `policygentemplates/common-ranGen.yaml` applies to all clusters with `common: true` set, `policygentemplates/group-du-sno-ranGen.yaml` applies to all clusters with `group-du-sno: ""` set.
 <5> Optional. The CR specifed under `KlusterletAddonConfig` is used to override the default `KlusterletAddonConfig` that is created for the cluster.
 <6> For single-node deployments, define a single host. For three-node deployments, define three hosts. For standard deployments, define three hosts with `role: master` and two or more hosts defined with `role: worker`.
-<7> BMC address that you use to access the host. Applies to all cluster types.
+<7> BMC address that you use to access the host. Applies to all cluster types. ZTP supports iPXE and virtual media booting by using Redfish or IPMI protocols. To use iPXE booting you must use RHACM 2.8 or later. For more information about BMC addressing, see the _Additional resources_ section. 
```

Comment:
I assume it make sense  Additional resources to be a link

Reply:
You would think so. But our policy is to not include links in the main body of content because it distracts the user from the task. So links are mostly in the **Additional resources** section only. And this text here just directs users to that section without using an actual link. (It's a controversial policy!)